packages/block-editor/src/components/url-input/index.js

```diff
@@ -253,7 +253,7 @@ class URLInput extends Component {
 					<Popover position="bottom" noArrow focusOnMount={ false }>
 						<div
 							className="editor-url-input__suggestions block-editor-url-input__suggestions"
-							id={ `editor-url-input-suggestions-${ instanceId }` }
+							id={ `block-editor-url-input-suggestions-${ instanceId }` }
```

Comment:
Can we extract this id to a variable and use it in both places to prevent the same issue to happen again in the future?

Reply:
Makes total sense! I extracted the id into a variable `suggestionsListboxId`. 👍